source/_posts/2024-05-09-music-assistant-2.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Music Assistant 2.0: Your Music, Your Players"
+description: "Connecting multiple music libraries to almost any smart speaker."
+date: 2024-05-09 00:00:01
+date_formatted: "May 09, 2024"
+author: Marcel van der Veldt
+comments: true
+categories: Music-Assistant
+og_image: /images/blog/2024-05-music-assistant/art.jpg
+---
+<p><img src='/images/blog/2024-05-music-assistant/art.jpg' class='no-shadow' /></p>
+
+Today, exactly five years ago, I, Marcel, started working on [Music Assistant](https://music-assistant.io/). What began as a quick script to sync my playlists when I wanted to switch between streaming providers grew into a beast on its own.
```

Comment:
```suggestion
Today, exactly five years ago, I, Marcel, started working on [Music Assistant](https://music-assistant.io/). What began as a quick script, sync my playlists so I could switch between streaming providers, grew into a beast on its own.
```

Reply:
Commas work, but it still needs the "to" before "sync"